client/app/scripts/utils/web-api-utils.js

```diff
@@ -225,8 +225,8 @@ export function getPipeStatus(pipeId) {
     },
     success: function(res) {
       const status = {
-        200: 'PIPE_ALIVE',
-        204: 'PIPE_DELETED'
+        204: 'PIPE_ALIVE',
+        404: 'PIPE_DELETED'
```

Comment:
@tomwilkie Just realized that the 404 check should probably be in the failure  branch (sorry I didn't catch this earlier). @foot Can you confirm?